modules/raveltechBidAdapter.js

```diff
@@ -0,0 +1,72 @@
+// Import the base adapter
+import { spec as baseAdapter } from './appnexusBidAdapter.js'; // eslint-disable-line prebid/validate-imports
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { logInfo } from '../src/utils.js';
+
+const BIDDER_CODE = 'raveltech';
+const URL = 'https://pb1.rvlproxy.net/bid/bid';
+// const URL_SIMPLE = 'https://pb1.rvlproxy.net/bid/simplebid';
+
+export const spec = {
+  code: BIDDER_CODE,
+  gvlid: baseAdapter.GVLID, // use base adapter gvlid
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} bidRequests A non-empty list of bid requests which should be sent to the Server.
+   * @return ServerRequest[] Info describing the requests to the server.
+   */
+  buildRequests: function(bidRequests, bidderRequest) {
+    if (!baseAdapter.buildRequests) { return []; }
+
+    let anonymizedBidRequests = baseAdapter.buildRequests(bidRequests, bidderRequest); // call the bid requests from the Appnexus adapter
+    if (!anonymizedBidRequests) { return []; } // if no bid request, return empty Array
+    if (!Array.isArray(anonymizedBidRequests)) { anonymizedBidRequests = [anonymizedBidRequests]; } // if only 1 bid request, anonymizedBidRequest will be an Object instead of an Array. Build Array with 1 bid request.
+
+    // Load ZKAD runtime, used to anonymize the uids
```

Comment:
if you are already loading js, why not just have your js call [registerBidAdapter ](https://github.com/prebid/Prebid.js/blob/40b906d17bcfda2b369924de2278ced73975edc2/src/prebid.js#L724) ?

Reply:
Alternatively, why not just modify the xandr bids in place with event listeners?